TeamCode/src/main/java/org/firstinspires/ftc/teamcode/robot/subsystems/Intake.java

```diff
@@ -0,0 +1,47 @@
+package org.firstinspires.ftc.teamcode.robot.subsystems;
+
+import com.qualcomm.robotcore.hardware.HardwareMap;
+import com.qualcomm.robotcore.hardware.Servo;
+
+import org.firstinspires.ftc.robotcore.external.Telemetry;
+import org.firstinspires.ftc.teamcode.core.lib.gamepad.GamepadManager;
+import org.firstinspires.ftc.teamcode.core.lib.interfaces.Subsystem;
+
+public class Intake implements Subsystem {
+    private static Intake instance;
+    private Servo servo;
+    private Telemetry telemetry;
+
+    private Intake() {
+    }
+
+    @Override
+    public void initialize(HardwareMap hardwareMap, Telemetry telemetry) {
+        servo = hardwareMap.get(Servo.class, "servoIntake");
+        servo.resetDeviceConfigurationForOpMode();
+    }
+
+    @Override
+    public void start() {
+
+    }
+
+    @Override
+    public void stop() {
+
+    }
+
+    @Override
+    public void execute(GamepadManager gamepadManager) {
+        servo.setPosition(1);
+        telemetry.addData("Intake Servo pos", servo.getPosition());
```

Comment:
sugestion to all:
add a default start to when adding telemetry data
"[INTAKE] + message"